packages/eslint-plugin/docs/rules/prefer-readonly.md

```diff
@@ -6,7 +6,7 @@ description: "Require private members to be marked as `readonly` if they're neve
 >
 > See **https://typescript-eslint.io/rules/prefer-readonly** for documentation.
 
-Member variables with the privacy `private` are never permitted to be modified outside of their declaring class.
+Private member variables (whether using the `private` modifier, or private fields) are never permitted to be modified outside of their declaring class.
```

Comment:
[Docs] Just a bit of nitpicking: the `,` is unnecessary I think, and for consistency + clarity it'd be good to show that `#` is the private fields thing.

```suggestion
Private member variables (whether using the `private` modifier or private `#` fields) are never permitted to be modified outside of their declaring class.
```